specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json

```diff
@@ -0,0 +1,422 @@
+                    }
+                },
+                "x-ms-pageable": {
+                "nextLinkName": null
+                }
+            }                
+        }
+    },
+    "definitions": {
+        "OperationListResult": {
+            "description": "Result of the request to list solution operations.",
+            "properties": {
+                "value": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/Operation"
+                },
+                "description": "List of solution operations supported by the OperationsManagement resource provider."
+                }
+            }
+        },
+        "Operation": {
+            "description": "Supported operation of OperationsManagement resource provider.",      
+            "properties": {
+                "name": {
+                    "description": "Operation name: {provider}/{resource}/{operation}",
+                    "type": "string"
+                },
+                "display": {
+                    "description": "Display metadata associated with the operation.",
+                    "properties": {
+                        "provider": {
+                            "description": "Service provider: Microsoft OperationsManagement.",
+                            "type": "string"
+                        },
+                        "resource": {
+                            "description": "Resource on which the operation is performed etc.",
+                            "type": "string"
+                        },
+                        "operation": {
+                            "description": "Type of operation: get, read, delete, etc.",
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "SolutionProperties": {
+            "description": "Solution properties supported by the OperationsManagement resource provider.",
+            "properties": {
+                "workspaceResourceId": {
+                    "type": "string",
+                    "description": "The azure resourceId for the workspace where the solution will be deployed/enabled."
+                },
+                "provisioningState": {
+                    "type": "string",
+                    "description": "The provisioning state for the solution.",
+                    "readOnly": true
+                },
+                "containedResources": {
+                    "type": "array",
+                    "items":{
+                      "type":"string",
+                      "description":"the azure resourceId of the resource."
+                    },
+                    "description": "The azure resources that will be contained within the solutions. They will be locked and gets deleted automatically when the solution is deleted."                    
+                },
+                "referencedResources": {
```

Comment:
add a property called resourceId for the items array.

Reply:
Why add this property? We have the Id property already.